Nos/AppDelegate.swift

```diff
@@ -24,7 +24,8 @@ class AppDelegate: NSObject, UIApplicationDelegate {
     ) {
         Task { 
             do {
-                try await PushNotificationService().registerForNotifications(with: deviceToken, user: currentUser) 
+                try await PushNotificationService().registerForNotifications(with: deviceToken, user: currentUser)
+                Log.info("PushNotifications: registered \(deviceToken.hexString)")
```

Comment:
Do we need this additional log, or was this just for testing?

Reply:
Oh no, I added that in order to test if the crash that happened to linda after logging out was related to receiving push notifications in the background. As it was also a modification of AppDelegate, I got confused and committed that to this PR which has noting to do with that. I just restored the file from main and fixed some incorrect end of lines and spacings it had.